roles/tackle/tasks/main.yml

```diff
@@ -3,6 +3,15 @@
   set_fact:
     api_groups: "{{ lookup('k8s', cluster_info='api_groups') }}"
 
+- name: Gather Facts
+  ansible.builtin.gather_facts:
+    gather_subset:
+      - user_uid
+
+- name: "Retrieve UID"
+  set_fact:
+    hub_task_uid: "{{ ansible_facts['user_uid'] }}"
```

Comment:
This is the hub UID, not the task UID, correct?  Perhaps `hub_uid` would be clearer?

Reply:
To be clear, when we retrieve this value we are getting it operator pod. The operator pod does not run as root, so it uses the default namespace UID.
```
$ oc exec -it mta-operator-cdf4858b7-7wch7 /bin/bash
kubectl exec [POD] [COMMAND] is DEPRECATED and will be removed in a future version. Use kubectl exec [POD] -- [COMMAND] instead.
bash-4.4$ id
uid=1000860000(1000860000) gid=0(root) groups=0(root),1000860000
```

We then set the hub to application container to run with this UID:
https://github.com/konveyor/operator/pull/395/files#diff-9e3528a228e3b18c37a6c6689149af80bcd315d76a2a3c2c0dd920e87fd6e07eR208

And pass it to the init container (which runs as root) so it can chown the files to the same UID:
https://github.com/konveyor/operator/pull/395/files#diff-9e3528a228e3b18c37a6c6689149af80bcd315d76a2a3c2c0dd920e87fd6e07eR223-R234

And then it gets passed down to the task pod as well (if not overridden with the now available environment variable).

I was trying to convey that this is the UID that will be used for the hub and task pods, but am open to changing the var name to something clearer/better if you have something in mind.

Reply:
didn't see `hub_uid` suggestion first time reading. Updated it to use this.